lib/workers/repository/updates/branchify.spec.ts

```diff
@@ -153,5 +153,38 @@ describe('workers/repository/updates/branchify', () => {
       expect(embedChangelogs).not.toHaveBeenCalled();
       expect(Object.keys(res.branches)).toHaveLength(2);
     });
+
+    it('fetch changelogs if required', async () => {
+      config.fetchReleaseNotes = true;
+      config.repoIsOnboarded = true;
+      mockedFunction(_changelog.needsChangelogs).mockReturnValueOnce(true);
+      flattenUpdates.mockResolvedValueOnce([
+        {
+          depName: 'foo',
+          branchName: 'foo',
+          prTitle: 'some-title',
+          version: '1.1.0',
+          groupName: 'My Group',
+          group: { branchName: 'renovate/{{groupSlug}}' },
+        },
+        {
+          depName: 'foo',
+          branchName: 'foo',
+          prTitle: 'some-title',
+          version: '2.0.0',
+        },
+        {
+          depName: 'bar',
+          branchName: 'bar-{{version}}',
+          prTitle: 'some-title',
+          version: '1.1.0',
+          groupName: 'My Group',
+          group: { branchName: 'renovate/my-group' },
+        },
+      ]);
+      const res = await branchifyUpgrades(config, {});
+      expect(embedChangelogs).toHaveBeenCalled();
```

Comment:
Maybe make number of calls more clear?

```suggestion
      expect(embedChangelogs).toHaveBeenCalledOnce();
```